playbooks/hbase-arm-build/run.yaml

```diff
@@ -0,0 +1,87 @@
+- hosts: all
+  become: yes
+  tasks:
+    - name: Build hbase
+      shell:
+        cmd: |
+          set -xe
+          # install java
+          apt-get update
+          apt install default-jre -y
+          apt install default-jdk -y
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-arm64
+          export PATH=$PATH:$JAVA_HOME/bin
+          echo "export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-arm64" >> ~/.profile
+          echo "export PATH=$PATH:$JAVA_HOME/bin" >> ~/.profile
+          source ~/.profile
+
+          # install mvn
+          apt install maven -y
```

Comment:
I suggest to download and config Maven from binary package with specific version, don't use apt, because some projects might depends on specific version, like: Spark, and please add a new role to do it